<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credits</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35,stop: 0 #fff, stop: 1 #bbb);
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Play"/>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qradialgradient(cx: 0.3, cy: -0.4,fx: 0.3, fy: -0.4,radius: 1.35,stop: 0 #fff, stop: 1 #bbb);
background-image: url('qt/back.png');</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tag1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Original design by Simon Rig√©t @ Coding Pirates 2019 &lt;img src=&quot;qt_windows/coding_pirates.png&quot;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Developed by Martin Kristensen @ Coding Pirates 2020 &lt;img src=&quot;qt_windows/coding_pirates.png&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
